docs/using-cinder-csi-plugin.md

```diff
@@ -206,6 +206,8 @@ storageclass.storage.k8s.io/csi-sc-cinderplugin created
 volumesnapshotclass.snapshot.storage.k8s.io/csi-cinder-snapclass created
 persistentvolumeclaim/pvc-snapshot-demo created
 ```
+Note: `force-create` can be used to force create the snapshot when the volume is `in-use` state.
```

Comment:
Please use a plain paragraph rather than a note:

```
Supported parameters for VolumeSnapshotClass:

* `force-create`: Support creating snapshot for a volume in in-use status.
```

Reply:
updated , tried several times and seems current position is best to add those info, thanks